jcl/src/openj9.jvm/share/classes/com/ibm/jvm/Log.java

```diff
@@ -67,22 +67,22 @@ public static int SetOptions(String options) {
 	 * directly.
 	 * @throws SecurityException
 	 */
-    private static void checkLegacySecurityPermssion() throws SecurityException {
-    	if (!("false".equalsIgnoreCase(com.ibm.oti.vm.VM.getVMLangAccess()	//$NON-NLS-1$
-    		.internalGetProperties().getProperty(LEGACY_LOG_PERMISSION_PROPERTY)))) {
-    		checkLogSecurityPermssion();
-    	}
-    }
+	private static void checkLegacySecurityPermssion() throws SecurityException {
+		if (!("false".equalsIgnoreCase(com.ibm.oti.vm.VM //$NON-NLS-1$
+				.internalGetProperties().getProperty(LEGACY_LOG_PERMISSION_PROPERTY)))) {
+			checkLogSecurityPermssion();
+		}
+	}
 	
```

Comment:
Since you're fixing some whitespace inconsistencies, could you also trim the trailing whitespace, please.

Reply:
Removed